File: lecture01/exercise01-B/solution.py
#!/usr/bin/env python3

'''
Exercise 01-B: Fixed Totals
'''

import sys

def main():
    while n := int(sys.stdin.readline()):
        total = sum(int(sys.stdin.readline()) for _ in range(n))
        print(total)

if __name__ == '__main__':
    main()

# vim: set sts=4 sw=4 ts=8 expandtab ft=python:

File: lecture01/exercise01-B/solution.cpp
// Exercise 01-B: Fixed Totals

#include <iostream>
#include <string>
#include <sstream>

using namespace std;

int main(int argc, char *argv[]) {
    int n, x;

    while (cin >> n && n) {
    	int total = 0;
    	for (int i = 0; i < n; i++) {
    	    cin >> x;
    	    total += x;
	}

	cout << total << endl;
    }

    return 0;
}

// vim: set sts=4 sw=4 ts=8 ft=cpp:
